gradle.properties

```diff
@@ -8,5 +8,6 @@ springBootVersion=2.1.1.RELEASE
 
 version=0.1.0-SNAPSHOT
 eventuateCommonImageVersion=0.10.0.RELEASE
-eventuateCdcImageVersion=0.7.0.RELEASE
-eventuateTramVersion=0.25.1.RELEASE
+eventuateMessagingKafkaImageVersion=0.9.0.RELEASE
+eventuateCdcImageVersion=local-test
```

Comment:
What's this?

Reply:
we don't have BUILD_SNAPSHOT versions for images, right? So I used local build, it should be changed when new version is released.